django_social_share/templates/django_social_share/templatetags/post_to_linkedin.html

```diff
@@ -0,0 +1,3 @@
+<div class="linkedin-this">
+    <a href="{{ linkedin_url|safe }}" target="_blank">{{link_text}}</a>
```

Comment:
I'd rather use `django.utils.safestring.mark_safe` in the template tag, so that template authors don't have to remember to add `|safe`